inst/input/hector_ssp119.ini

```diff
@@ -138,6 +138,7 @@ CNOX=0.0042			; coefficent for NOX
 CCO=-0.000105		; coefficent for CO
 CNMVOC=-0.000315	; coefficent for NMVOC (non methane VOC)
 CCH4=-0.32			; coefficent for CH4
+CH2=-0.00044625 ; coefficent for H2
```

Comment:
Same spelling note; also, is there a reference/source for this value? If so please note it.

Reply:
Was there some derivation needed for this, or was this coefficient in the right format from the original source? (And how similar were the coefficients for other emissions to the ones we are using now?)

Reply:
Okay there was a derivation that was needed for this parameter, which will be described in detail in the manuscript associated with this work although that is a TBD, so I have included a TODO-H2 tag which is linked to the H2 project as a reminder to address this issue before merging into main or anything. 

@ssmithClimate Could you elaborate on this a bit more? (And how similar were the coefficients for other emissions to the ones we are using now?) I am not sure if I understand what you are asking here. It is also unclear where the original coefficients came from @ssmithClimate do you know the source of those?

Reply:
Sorry for the delay. The coefficients in hector now ultimately derive from this paper:
Wigley, T.M.L., Steven J. Smith, and M.J. Prather (2002) Radiative Forcing due to Reactive Gas Emissions. Journal of Climate 15(18), pp. 2690–2696

Although the hector paper quotes the source as :
Tanaka, K., Kriegler, E., Bruckner, T., Hooss, C., Knorr, W., and Raddatz, T.: Aggregated Carbon Cycle, Atmospheric Chemistry, and Climate Model (ACC2) – description of the forward and inverse models, 1–188, Max Planck Institute for Meteorology, Hamburg, Germany, 188, 2007.
(which is a more indirect reference)

So my question was - is the overall chemistry representation of the source we're drawing the H2 coefficients from is generally consistent with the chemistry representation used now in Hector. If not it's something to flag (e.g., means we ultimately might need to update more than H2) since the two things we're merging together are not entirely consistent with each other. (Given uncertainty its likely fine, but we would want to flag this at least.)

So the way I thought to evaluate that was to look if the relative magnitude of any common coefficients for the non-H2 components of the equations that were used in the H2 work we're using now and the original equation we use now in Hector were similar. For example is the ratio of the impact of NOx to impact of CH4 or NOx to CO in the original hector (e.g. Wigley et al, Tanaka et al) are similar to what's in the H2 papers we are citing (to the extent they contain these other effects).

For example, in hector we have something like this for the CH4 OH lifetime (Hartin et al. 2015):

ln(OH)t = −0.32(ln[CH4 ]t − ln[CH4 ]t 0 ) + 0.0042(E(NOx)t ) − (E(NOx)t0) − 0.000105(E(CO)t ) − (E(CO)t0) − 0.00315(E(VOC)t ) − (E(VOC)t0)

So the impact of NOx vs CO is: 0.0042 / 0.000105 = 40 times (on an emissions basis in this case). Or we could look at the relative impact of NOx to CH4 in this equation if that's what's common between what's in hector now and what is represented in the equations from the paper's we are drawing from for the new H2 components.

To do this we need to have both sets of equations cast into an equivalent form.